### PR TITLE
feat: base for agent app

### PR DESCRIPTION
**Pull request by 0xyaco**
# 🤖 Linear

Closes GRT-167

## Description
Set up the basic configuration for the main `agent` app.

### CODE REVIEW

**Comment by 0xyaco** [comment]
@jahabeebs totally! Docs, readme, licensing and stuff like that is going to be tackled by the end of the project. But overall, yeah, given the complexity of the project, docs must be super detailed and complete so they are a **must**.

**Review by 0xkenj1 — DISMISSED** [review]
LGTM after addressing @jahabeebs comments :)

File: apps/agent/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "agent",
+    "version": "1.0.0",
+    "description": "",
```
[the remark this answers is not in the document]
Comment (0xyaco) [reply apps/agent/package.json:4]: Added the `private` property, need to ask for The Graph team about licensing.

**Review by 0xkenj1 — APPROVED** [review]

File: apps/agent/package.json
Comment (0xnigir1) [inline apps/agent/package.json:1]: would you need a `dev` script for running it with hot reload? [`tsx`](https://tsx.is/) is the one we used in chainhub

File: packages/automated-dispute/package.json [hunk 2]
```diff
@@ -2,8 +2,9 @@
     "name": "@ebo-agent/automated-dispute",
     "version": "1.0.0",
     "description": "",
-    "main": "index.ts",
+    "main": "./dist/index.js",
     "type": "module",
+    "private": "true",
```
Comment (0xnigir1) [inline packages/automated-dispute/package.json:7]:
```suggestion
    "types": "./dist/index.d.ts",
    "files": [
        "dist/*",
        "package.json",
        "!**/*.tsbuildinfo"
    ],
    "private": "true",
```

File: packages/blocknumber/package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
     "description": "",
     "main": "./dist/index.js",
     "type": "module",
+    "private": "true",
```
Comment (0xnigir1) [inline packages/blocknumber/package.json:7]:
```suggestion
    "types": "./dist/index.d.ts",
    "files": [
        "dist/*",
        "package.json",
        "!**/*.tsbuildinfo"
    ],
    "private": "true",
```

File: packages/shared/package.json [hunk 4]
```diff
@@ -4,6 +4,7 @@
     "description": "",
     "main": "./dist/index.js",
     "type": "module",
+    "private": "true",
```
Comment (0xnigir1) [inline packages/shared/package.json:7]:
```suggestion
    "types": "./dist/index.d.ts",
    "files": [
        "dist/*",
        "package.json",
        "!**/*.tsbuildinfo"
    ],
    "private": "true",
```

**Review by 0xnigir1 — DISMISSED** [review]
lgtm, left some suggestions (not critical) but i'm not sure if the paths I specified are correct

**Review by 0xnigir1 — APPROVED** [review]

**Pull request merged**